benchmarks/cudf-merge.py

```diff
@@ -4,23 +4,35 @@
 import argparse
 import asyncio
 import cProfile
+import gc
 import io
+import os
 import pickle
 import pstats
-import sys
 from time import perf_counter as clock
 
 import cupy
 import numpy as np
 
 from dask.utils import format_bytes, format_time
 
-import cudf
-import rmm
-
 import ucp
+from ucp._libs.utils import print_multi, print_separator
 from ucp.utils import run_on_local_network
 
+# Must be set _before_ importing RAPIDS libraries (cuDF, RMM)
+os.environ["RAPIDS_NO_INITIALIZE"] = "True"
+
+
+import cudf  # noqa
+import rmm  # noqa
```

Comment:
Why do we need the noqa? Is it because flake8 complains about imports not at top-level?

Reply:
It actually is `isort` rather than `flake8`, but yes.